notebooks.go

```diff
@@ -363,6 +368,88 @@ func (s *server) GetNotebooks(w http.ResponseWriter, r *http.Request) {
 	s.respond(w, r, resp)
 }
 
+func (s *server) GetDefaultNotebook(w http.ResponseWriter, r *http.Request) {
+	vars := mux.Vars(r)
+	namespace := vars["namespace"]
+
+	log.Printf("loading notebooks for %q", namespace)
+
+	notebooks, err := s.listers.notebooks.Notebooks(namespace).List(labels.Everything())
+	if err != nil {
+		s.error(w, r, err)
+		return
+	}
+
+	resp := &notebookapiresponse{
+		APIResponseBase: APIResponseBase{
+			Success: false,
+			Status:  http.StatusOK,
+		},
+	}
+
+	for _, notebook := range notebooks {
+
+		if val, ok := notebook.Labels["notebook.statcan.gc.ca/default-notebook=true"]; ok {
+			if val == "true" {
```

Comment:
What is this for?

Reply:
It's me forgetting to remove the second if while i was testing